connection.go

```diff
@@ -324,8 +325,10 @@ func (c *connection) login(ctx context.Context) error {
 		},
 	}
 
-	if osUser, err := user.Current(); err != nil {
+	if osUser, err := user.Current(); err == nil && osUser != nil {
 		authRequest.ClientOsUsername = osUser.Username
+	} else {
+		logCouldNotGetOsUser(err)
```

Comment:
wouldn't you create an error and log it using the standard logging facitly? Also don't realy get why this is a fix, the error is just reported not realy dealt with.

Reply:
I am not sure what else to do in this case.